contrib/node/src/python/pants/contrib/node/tasks/node_resolve.py

```diff
@@ -4,20 +4,37 @@
 
 from __future__ import absolute_import, division, print_function, unicode_literals
 
+import os
+
+from pants.base.build_environment import get_buildroot
 from pants.base.workunit import WorkUnitLabel
+from pants.build_graph.build_graph import sort_targets
 
-from pants.contrib.node.tasks.node_paths import NodePaths
+from pants.contrib.node.tasks.node_paths import NodePaths, NodePathsLocal
 from pants.contrib.node.tasks.node_task import NodeTask
 
 
 class NodeResolve(NodeTask):
-  """Resolves node_package targets to isolated chroots using different registered resolvers."""
+  """Resolves node_package targets to their node paths using different registered resolvers.
+
+  This task exposes two products NodePaths and NodePathsLocal. Both products are handled
+  optionally allowing the consumer to choose.
+
+  NodePaths contain a mapping of targets and their resolved location in the virtualized
+  pants working directory.
+
+  NodePathsLocal is similar to NodePaths with the difference that the resolved location
+  is local to the target source root.
```

Comment:
"local to" doesn't have much meaning... could just say "next to", or refer explicitly to "the directory that the target is defined in".